### PR TITLE
README: Add custom gitlab examples

### PR DESCRIPTION
**Pull request by Ankirama**
Fixed issue #66 by adding gitlab examples in README.md

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -45,9 +45,14 @@ sao user/repo --clone
 sao gitlab:user/repo
 sao bitbucket:user/repo
 
+# Use custom git host
+sao gitlab:gitlab.mydomain.com:user/repo
+sao gitlab.mydomain.com:user/repo
```
Comment (egoist) [inline docs/README.md]: this doesn't work I guess 😄

**Comment by egoist** [comment]
I'm actually try to make SAO support git url directly :D like below:

```bash
sao https://domain/path/to.git
sao git@domain.com/to.git
```

File: docs/README.md (hunk 1 above)
Comment (Ankirama) [reply docs/README.md]: I tried it with my custom gitlab and ssh key :)
`sao gitlab.weezyu.com:weezyu/api --clone`
`sao gitlab:gitlab.weezyu.com:weezyu/api`

You can find these informations here: https://www.npmjs.com/package/download-git-repo

**Comment by Ankirama** [comment]
So you want to stop using download-git-repo ? 
Nice 👍

**Comment by egoist** [comment]
oh seems download-git-repo supports custom gitlab host natively but SAO broke that 😅 will fix this too.

**Comment by Ankirama** [comment]
Your 0.22.5 isn't broken :)
Here my output:
```
$> sao gitlab:gitlab.weezyu.com:weezyu/api                                                                                                          warn Config file was not found, we simply copy it to dest folder!
success Done, let the hacking begin!
```

And I got my files without any issue.

**Comment by egoist** [comment]
Indeed it's working 😅

**Pull request merged**